requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ pytest-localftpserver
 coverage
 sphinx==2.2.1
 sphinx_rtd_theme==0.4.3
+paramiko
```

Comment:
```suggestion
```

Not necessary since we're trying to test without paramiko initially and only install it on particular jobs.